From 6ff8bda992463a8c0dc34ee18820ca56cf9cccc0 Mon Sep 17 00:00:00 2001
From: Yegor Yefremov <yegorslists@googlemail.com>
Date: Wed, 30 Mar 2022 10:11:16 +0200
Subject: [PATCH] hostapd: Add the missing CONFIG_SAE option to the defconfig

CONFIG_SAE was added to wpa_supplicant's defconfig but wasn't
added to the hostapd's defconfig file.

Signed-off-by: Yegor Yefremov <yegorslists@googlemail.com>
---
 hostapd/defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/hostapd/defconfig b/hostapd/defconfig
index 6b50b6c59..611f96ffa 100644
--- a/hostapd/defconfig
+++ b/hostapd/defconfig
@@ -161,6 +161,9 @@ CONFIG_IPV6=y
 # final IEEE 802.11ax version.
 #CONFIG_IEEE80211AX=y
 
+# Simultaneous Authentication of Equals (SAE), WPA3-Personal
+#CONFIG_SAE=y
+
 # Remove debugging code that is printing out debug messages to stdout.
 # This can be used to reduce the size of the hostapd considerably if debugging
 # code is not needed.
-- 
2.30.2

